src/com/smilingrob/plugin/robohexar/ModelError.java

```diff
@@ -42,11 +42,11 @@ public PsiElement getElementToTag() {
     public String messageForError() {
         switch (mType) {
             case ACCESSORS_NOT_ALPHABETIZED:
-                return "Accessors not alphabetized in class.";
+                return "Hey dummy, accessors are not alphabetized in this class.";
```

Comment:
Wow it's fully accusatory. I don't think I've even seen Hexar say stuff like that, even jokingly.